source/stylesheets/refills/_phone.scss

```diff
@@ -0,0 +1,68 @@
+.phone {
+  $medium-screen: em(640) !default;
+  $large-screen: em(860) !default;
+  $ratio-small: 1;
+  $ratio-large: 1.6;
+  $inset-shadow: inset 0 1px 4px transparentize(#000, 0.8);
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.